src/masternodes/rpc_proposals.cpp

```diff
@@ -612,7 +613,7 @@ UniValue votegovbatch(const JSONRPCRequest &request) {
     RPCTypeCheck(request.params, {UniValue::VARR}, false);
 
     const auto &keys = request.params[0].get_array();
-    auto sleepTime = request.params[1].isNull() ? SLEEP_TIME_MILLIS : request.params[1].get_int();
+    auto sleepTime = (request.params.size() > 1 && request.params[1].isNull()) ? SLEEP_TIME_MILLIS : request.params[1].get_int();
```

Comment:
You can leave this as it but isNull() is safe to call without a size() check as UniValue defines [] usage and makes it safe to use on non-existent entries. Check other isNull() usage in the code on optional RPC arguments.